gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/KubernetesPlugin.java

```diff
@@ -38,13 +39,15 @@ public KubernetesPlugin() {
   public Map<String, Collection<Class<? extends Task>>> getTaskPrecedence() {
     final Map<String, Collection<Class<? extends Task>>> ret = new HashMap<>();
     ret.put("k8sApply", Collections.singletonList(KubernetesResourceTask.class));
+    ret.put("k8sPush", Collections.singletonList(KubernetesBuildTask.class));
     return ret;
   }
 
   @Override
   protected void jKubeApply(Project project) {
     register(project, "k8sConfigView", KubernetesConfigViewTask.class);
     register(project, "k8sBuild", KubernetesBuildTask.class);
+    register(project, "k8sPush", KubernetesPushTask.class);
```

Comment:
Could you add the appropriate entry for the SimpleIT#tasks_containsKubernetesAndOpenShiftTasks test?